recipes/bacpage/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set version = "2023.11.10.1" %}
+
+package:
+  name: bacpage
+  version: {{ version }}
+
+source:
+  url: https://github.com/CholGen/bacpage/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 7a62278131407195a8f7555a92ed2d86ca7e63366bac6d280dac337e5f5218c5 
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install --no-deps --ignore-installed ."
+  run_exports:
+    - {{ pin_subpackage('bacpage') }}
+
+requirements:
+  host:
+    - python <3.12,>=3.9
+    - pip
+  run:
+    - python <3.12,>=3.9
+    - bc
+    - bcftools =1.17
```

Comment:
are those strict pins really necessary? Can they be relaxed?

Reply:
Python <3.12 is required as snakemake has an issue with the updates to f-strings in 3.12. 3.9 is just there because the solver once settled on python ~2.7, and I would like to avoid that.

bcftools v1.17 is required because I use flags added in that version.

Reply:
But there is more, bedtools, bwa, emboss ... I can only recommend to not pin that strictly.

Reply:
@watronfire ping

Reply:
My thinking was that I know this set of packages and versions is able to be solved by conda/mamba and enables my pipeline to run correctly. And I wasn't sure how well the pipeline can tolerate deviations from it. 

Anyway, the program tests successfully when the pins are removed from everything from python, samtools, and bcftools (which have to be pinned for the reasons above).